### PR TITLE
chore(agw): `magma_trfserver` is upgraded to Ubuntu 20.04

### LINKED ISSUE #13203: [Enhancement] TRF Server VM should be migrated to Ubuntu 20.04
### Issue Description:
The TRF Server VM still contains Debian 9 OS. The OS version for TRF Server should be migrated to Ubuntu 20.04 to keep software versions updated and in sync with the Magma-Dev and Magma-Test VMs. The primary reason is that some of the traffic server code is being shared between Magma-Test and TRF Server VMs, and hence, upgradation of software packages or adapted coding style in one VM may not be applied in other and may lead to temporary failures.

One such recent examples of issue with python version requirement can be seen with PRs https://github.com/magma/magma/pull/12938 and https://github.com/magma/magma/pull/13202

### LINKED ISSUE #13245: magma_trfserver not starting up correctly on Mac OS
### Your Environment

- **Version:** master
- **Affected Component:** magma_trfserver
- **Deployment Environment:** Vagrant 2.2.19, Virtualbox 6.1.34 r150636, MacOS 11.6.5

### Describe the Issue

When running `vagrant up magma_trfserver` after destroying it (`vagrant destroy magma_trfserver`) on MacOS (but not on Linux), the process fails with the following error

```
==> magma_trfserver: Configuring and enabling network interfaces...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

/sbin/ifup 'eth1'
/sbin/ifup 'eth2'
/sbin/ifup 'eth3'

Stdout from the command:



Stderr from the command:

ifup: interface eth1 already configured
ifup: interface eth2 already configured
RTNETLINK answers: File exists
ifup: failed to bring up eth3
```

Repeating the `vagrant up magma_trfserver` finishes the provisioning but cannot set the symlink to traffic_server.py because the file does not exist.

```
TASK [trfserver : Create symlink for trfserver scripts] ************************
Friday 08 July 2022  11:40:00 +0200 (0:00:00.019)       0:00:04.219 *********** 
changed: [magma_trfserver] => (item=traffic_server.py) => {"ansible_loop_var": "item", "changed": true, "dest": "/usr/local/bin/traffic_server.py", "item": "traffic_server.py", "src": "/home/vagrant/magma/lte/gateway/deploy/roles/trfserver/files/traffic_server.py", "warnings": ["Cannot set fs attributes on a non-existent symlink target. follow should be set to False to avoid this."]}
[WARNING]: Cannot set fs attributes on a non-existent symlink target. follow
should be set to False to avoid this.
``` 

**Expected behavior**
should run without error, as it does on Linux. (Or when reverting commit #13010 locally)

### PR DESCRIPTION
**Pull request by mpfirrmann**
## Summary

The packed VM is already uploaded to the Vagrant Cloud (https://app.vagrantup.com/magmacore/boxes/magma_trfserver/versions/1.1.20220722). This PR contains the following:

- The `magma_trfserver`-entry in the `Vagrantfile` is adapted to the latest version and more memory is allocated due to higher demands on Ubuntu focal.
- The names of the cached boxes in the GitHub workflows are adapted to the box version in the `Vagrantfile`.
- The Ansible playbook of the traffic server is adapted to Ubuntu focal. The Debian related roles are dropped.
- The underlying `iperf3` package is bumped to the latest version.
- The magma VMs still use the "classical" naming scheme for the network adapters, we do rename them in them in the preburn images (more information e.g. [HERE](https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/)). The adapters are renamed here as well.
- The source of the corresponding `iperf3-python` package is adapted to the original and newer GitHub repo.
- The command for starting the `magma_trfserver` in the CWF integration tests is adapted.
- The `.yml` for packing the trfserver VM is adapted to Ubuntu focal.
- The general packing process of Ubuntu focal machines is improved with respect to box size and the respective license files for the code's origins are added.

This PR closes https://github.com/magma/magma/issues/13203.

## Test Plan

All integration tests were run locally and via GitHub actions on my fork.

LTE integration tests: https://github.com/mpfirrmann/magma/actions/runs/2739321180
CWF integration tests: https://github.com/mpfirrmann/magma/actions/runs/2742121590
Federated integration tests: https://github.com/mpfirrmann/magma/actions/runs/2739508914

## Additional Information

This PR will also ...

- ... most certainly fix https://github.com/magma/magma/issues/13245 and with it unblock https://github.com/magma/magma/issues/13337
- ... enable the merge (after the revert) of https://github.com/magma/magma/pull/12938. The traffic server now runs Python 3.8 as well.
- ... unblock further updates of python dependencies as in https://github.com/magma/magma/pull/12972 and https://github.com/magma/magma/issues/13426

### CODE REVIEW

**Review by crasu — APPROVED** [review]

**Review by ajahl — APPROVED** [review]

File: lte/gateway/deploy/roles/trfserver/files/magma_interfaces [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# interfaces(5) file used by ifup(8) and ifdown(8)
```
Comment (maxhbr) [inline lte/gateway/deploy/roles/trfserver/files/magma_interfaces:12]: maybe add a license header here

**Review by maxhbr — COMMENTED** [review]
maybe move `orc8r/tools/packer/scripts/third_party` to `orc8r/tools/packer/scripts/third_party/${PROJECT_NAME}`?

File: lte/gateway/Vagrantfile [hunk 2]
```diff
@@ -75,7 +75,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     magma_trfserver.vm.provider "virtualbox" do |vb|
       vb.name = "magma-trfserver"
       vb.linked_clone = true
-      vb.customize ["modifyvm", :id, "--memory", "256"]
+      vb.customize ["modifyvm", :id, "--memory", "1024"]
```
Comment (maxhbr) [inline lte/gateway/Vagrantfile:78]: why is that needed?

**Review by maxhbr — APPROVED** [review]
LGTM up to minor comments

**Review by Neudrino — APPROVED** [review]

**Review by LKreutzer — APPROVED** [review]

File: lte/gateway/Vagrantfile (hunk 2 above)
Comment (mpfirrmann) [reply lte/gateway/Vagrantfile:78]: Ubuntu does need more memory. The machine did not even start with 256.

File: lte/gateway/deploy/roles/trfserver/files/magma_interfaces (hunk 1 above)
Comment (mpfirrmann) [reply lte/gateway/deploy/roles/trfserver/files/magma_interfaces:12]: This file is copied into the `/etc/network/interfaces` on the traffic server (as a identical copy of an Ansible step in the `dev_common` role). Header was added.

**Review by nstng — APPROVED** [review]
lgtm - mob review
